test/Microsoft.ML.OnnxTransformTest/OnnxTransformTests.cs

```diff
@@ -0,0 +1,215 @@
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using System.Runtime.InteropServices;
+using Microsoft.ML.OnnxScoring;
+
+namespace Microsoft.ML.Tests
+{
+    public class OnnxTransformTests : TestDataPipeBase
+    {
+
+        private const int inputsize = 150528;
+
+        private class TestData
+        {
+            [VectorType(inputsize)]
+            public float[] data_0;
+        }
+        private class TestDataSize
+        {
+            [VectorType(2)]
+            public float[] data_0;
+        }
+        private class TestDataXY
+        {
+            [VectorType(inputsize)]
+            public float[] A;
+        }
+        private class TestDataDifferntType
+        {
+            [VectorType(inputsize)]
+            public string[] data_0;
+        }
+
+        private float[] getSampleArrayData()
+        {
+            var samplevector = new float[inputsize];
+            for (int i = 0; i < inputsize; i++)
+                samplevector[i] = (i / (inputsize * 1.01f));
+            return samplevector;
+        }
+
+        public OnnxTransformTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        void TestSimpleCase()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                return;
+            var modelFile = "squeezenet/00000001/model.onnx";
+
+            var samplevector = getSampleArrayData();
+
+            var dataView = ComponentCreation.CreateDataView(Env,
+                new List<TestData>(new TestData[] {
+                    new TestData()
+                    {
+                        data_0 = samplevector
+                    },
+                     new TestData()
+                     {
+                    data_0 = samplevector
+                     }
+                }));
+
+            var xyData = new List<TestDataXY> { new TestDataXY() { A = new float[inputsize] } };
+            var stringData = new List<TestDataDifferntType> { new TestDataDifferntType() { data_0 = new string[inputsize] } };
+            var sizeData = new List<TestDataSize> { new TestDataSize() { data_0 = new float[2] } };
+            var pipe = new OnnxEstimator(Env, modelFile, "data_0", "softmaxout_1");
+
+            var invalidDataWrongNames = ComponentCreation.CreateDataView(Env, xyData);
+            var invalidDataWrongTypes = ComponentCreation.CreateDataView(Env, stringData);
+            var invalidDataWrongVectorSize = ComponentCreation.CreateDataView(Env, sizeData);
+            TestEstimatorCore(pipe, dataView, invalidInput: invalidDataWrongNames);
+            //TestEstimatorCore(pipe, dataView, invalidInput: invalidDataWrongTypes);
+
+            pipe.GetOutputSchema(SchemaShape.Create(invalidDataWrongVectorSize.Schema));
+            try
+            {
+                pipe.Fit(invalidDataWrongVectorSize);
+                //Assert.False(true);
```

Comment:
>//Assert.False(true); [](start = 16, length = 21)

why this is commented?

Reply:
Uncommented now.

---
In reply to: [220398186](https://github.com/dotnet/machinelearning/pull/942#discussion_r220398186) [](ancestors = 220398186)